plugins/collaajUpdatePlugin/CollaajUpdatePlugin.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+class CollaajUpdatePlugin extends KalturaPlugin implements IKalturaServices {
+
+    const PLUGIN_NAME = "CollaajUpdate";

```

Comment:
Also, we agreed to call it captureSpace.
@gonenradai, please correct me if I'm wrong.